tests/Numerics/MatrixTest.php

```diff
@@ -1,93 +1,89 @@
-<?php
-require_once 'PHPUnit/Framework.php';
-require_once 'PHPUnit/TextUI/TestRunner.php';
+<?php namespace Moserware\Skills\Tests\Numerics;
 
-require_once(dirname(__FILE__) . '/../../Skills/Numerics/Matrix.php');
+use Moserware\Skills\Numerics\IdentityMatrix;
+use Moserware\Skills\Numerics\Matrix;
+use Moserware\Skills\Numerics\SquareMatrix;
+use Moserware\Skills\Tests\TestCase;
 
-use \PHPUnit_Framework_TestCase;
-use Moserware\Numerics\Matrix;
-use Moserware\Numerics\IdentityMatrix;
-use Moserware\Numerics\SquareMatrix;
-
-class MatrixTest extends PHPUnit_Framework_TestCase
+class MatrixTest extends TestCase
 {
     public function testTwoByTwoDeterminant()
     {
         $a = new SquareMatrix(1, 2,
-                              3, 4);
+            3, 4);
```

Comment:
This appears not to line up like it used to (e.g. the 3 under the 1 to make it look like a square matrix)